src/node_v8.cc

```diff
@@ -389,6 +389,18 @@ void GCProfiler::Stop(const FunctionCallbackInfo<v8::Value>& args) {
                                 .ToLocalChecked());
 }
 
+void GetMaxYoungGenerationSize(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  size_t value = env->isolate_data()->max_young_gen_size;
```

Comment:
This isn’t going to return a meaningful value for embedded scenarios (ditto below)